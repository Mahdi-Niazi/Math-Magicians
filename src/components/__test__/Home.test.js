import renderer from 'react-test-renderer';
import Home from '../../pages/Home';

it('Snapshot of the home page', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
