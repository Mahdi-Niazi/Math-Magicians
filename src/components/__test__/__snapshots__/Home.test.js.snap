// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of the home page 1`] = `
<div
  className="Home-txt"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin porta nisl ac eros cursus tincidunt vel faucibus lacus. Vivamus id vulputate lectus. Donec neque risus, varius sed pulvinar vel, ultricies at augue. Duis consequat odio sed tempus auctor. Nam efficitur nibh quis ornare vulputate. Nulla eget tortor id justo sollicitudin mattis nec eget eros. Integer ut vulputate velit. Quisque nec gravida lectus. Duis at tempor ipsum. Duis lectus urna, aliquet ac quam in, imperdiet bibendum dolor. Sed a nisl semper, pharetra justo quis, aliquet neque. Nunc vestibulum imperdiet lorem ac malesuada. Suspendisse vitae ante elementum, hendrerit dolor sed, convallis est. Morbi et nisl magna. Nam urna est, molestie in orci ut, lobortis pharetra magna. Etiam mattis nisl tellus, vestibulum tristique neque eleifend eu. Integer at eleifend augue, vitae maximus nisi. Vivamus tincidunt erat eros, a fringilla dui eleifend convallis. Morbi rhoncus orci neque, a tristique ligula tempus quis.
    <p>
      Sed nec orci efficitur magna vestibulum gravida quis pellentesque diam. Suspendisse libero nibh, lacinia vitae dui ac, vulputate aliquam metus. Etiam vel mauris nulla. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nulla metus neque, elementum sit amet tellus ac, sollicitudin tempus tortor. Aenean imperdiet laoreet bibendum. Quisque eget eros neque. Maecenas aliquam ullamcorper faucibus.
    </p>
    Aenean tortor nibh, consectetur nec placerat eu, fringilla vitae nulla. Cras posuere metus mi, quis condimentum risus semper et. Cras ut elit eleifend, dapibus lacus vitae, gravida metus. Vestibulum pretium tortor sit amet tortor lobortis egestas. Curabitur eget mi magna.
  </p>
</div>
`;
